packages/ui-stencil/src/components/internal/orama-modal/orama-modal.tsx

```diff
@@ -14,9 +14,10 @@ export class OramaModal {
   @Prop() closeOnEscape = true
   @Prop() closeOnOutsideClick = true
   @Prop() mainTitle = ''
+  @Prop() layout?: 'modal' | 'embed' = 'modal'
```

Comment:
not sure about this prop here as `orama-modal` component should behave always as modal by nature. I would avoid adding this prop that is needed by the parent component only

Reply:
I've created a `orama-embed` layout to avoid handling this different behavior inside the modal